sklearn/mixture/tests/test_gmm.py

```diff
@@ -471,3 +470,33 @@ def test_verbose_second_level():
         g.fit(X)
     finally:
         sys.stdout = old_stdout
+
+def test_GMM_deprecation_warnings():
+    assert_warns_message(DeprecationWarning,
+                         "The 'GMM' class has been renamed to "
+                         "'GaussianMixtureModel' in 0.17 and "
+                         "will be removed in 0.19. "
+                         "Use the GaussianMixtureModel instead.", mixture.GMM)
+    assert_warns_message(DeprecationWarning,
+                         "'params' has been deprecated in 0.17 "
+                         "and will be removed in 0.19.", mixture.GMM,
+                         params='wm')
+    g = mixture.GMM()
+    mean = [0, 0]
+    cov = [[1, 0], [0, 1]]
+    X = rng.multivariate_normal(mean, cov, 100)
+    g.fit(X)
+    assert_warns_message(DeprecationWarning,
+                         "The 'score' method in GMM has been deprecated "
+                         "in 0.17 and will be removed in 0.19. "
+                         "Use 'score_samples' method "
+                         "in GaussianMixtureModel instead.", g.score, X)
+    assert_warns_message(DeprecationWarning,
+                         "The 'score_samples' method in GMM has been "
+                         "deprecated in 0.17 and will be removed in 0.19."
+                         "Use 'score_samples' method "
```

Comment:
*the* 'score_samples' method, *the* 'log_probability', *the* 'predict_proba' method, *the* 'responsibility'. Same in all deprecation warnings.

Also I think '*of* each sample' sounds better than '*for* each sample' in this context.